### PR TITLE
Reducing the scope of the variable ret

### PR DESCRIPTION
**Pull request by prakritigoyal19**
This pr fixes issue #329

### CODE REVIEW

**Comment by Snorch** [comment]
The second commit looks like an excess leftover https://github.com/checkpoint-restore/criu/pull/1065/commits/ce12966321a0654d3699f87e134a105c7777304d Can you please remove it? (Or am I missing something?)

We have actually a lot of places (for instance find_remap_mounts and variable m) where we have the scope of variable wider when it is required. Not sure if we really need to fix it, because fixing it everywhere might be a pain, except the case if it is required for some of your future changes, is it?

**Comment by prakritigoyal19** [comment]
No, these changes are currently not required to the future changes that I am going to make. I saw this as an open issue and while reading through the code, found these variables at a wider scope and hence decided to send this pr.

@xemul  in the open issue #329 were the expectations different than what has been submitted? If the expectations were not the same, please let me know what to change and I'll try to fix it to what was expected.
 
Else if these changes are not needed anymore, should I close the pr?

**Comment by xemul** [comment]
@prakritigoyal19 , no your pr is OK, but the 2nd commit titled "Merge remote-tracking branch 'upstream/criu-dev' into enhance" looks excessive.

As far as "fixing more places" from @Snorch is concerned -- yes, there are many other places to fix, if you feel courage, you may include them too, but even without it the pr worth merging.

**Comment by prakritigoyal19** [comment]
Done!

I have removed the second commit. I will also try to reduce scope of the other variables too and send them in the near future.

**Comment by prakritigoyal19** [comment]
I have moved the ret declaration deeper. Thank you for pointing it out.

I will also try to make the commit message more descriptive in the future. Sorry for the confusion caused!

**Comment by avagin** [comment]
```
commit 98048d0eef70e6a28b0a4f0f29c1cefe86226a31
Author: prakritigoyal19 <prakritigoyal19@gmail.com>
Date:   Sat May 23 09:41:12 2020 +0530

    Fixes #329: reduce scope of the variable ret (2)
    
    Signed-off-by: prakritigoyal19

commit fde00e643c770951156fa269cdf9165be76500fe
Author: prakritigoyal19 <prakritigoyal19@gmail.com>
Date:   Tue May 19 15:27:01 2020 +0530

    reduce scope of integer variable ret
    
    Signed-off-by: prakritigoyal19
```

It will be cleaner if you merge these two commits. " Fixes #329:" has to be moved from the commit subject to the commit body.

File: criu/files-reg.c [hunk 1]
```diff
@@ -155,15 +155,15 @@ static int trim_last_parent(char *path)
 static int copy_chunk_from_file(int fd, int img, off_t off, size_t len)
 {
 	char *buf = NULL;
-	int ret;
-
+
```
Comment (Snorch) [inline criu/files-reg.c:158]:
```
s/\s*\s$//
```
Please remove spaces in the end of line.

File: test/zdtm/static/socket-tcpbuf.c [hunk 2]
```diff
@@ -45,8 +45,7 @@ static int fill_sock_buf(int fd)
 {
 	int flags;
 	int size;
-	int ret;
-
+
```
Comment (Snorch) [inline test/zdtm/static/socket-tcpbuf.c:48]: The same.

File: criu/files-reg.c [hunk 3]
```diff
@@ -155,15 +155,14 @@ static int trim_last_parent(char *path)
 static int copy_chunk_from_file(int fd, int img, off_t off, size_t len)
 {
 	char *buf = NULL;
-	int ret;
-
```
Comment (Snorch) [inline criu/files-reg.c:159]: Please don't remove newline between variable declaration and next block of code.

Comment (prakritigoyal19) [reply criu/files-reg.c:159]: @Snorch I have made these changes. Please let me know if there any other changes to be made.

**Comment by xemul** [comment]
I guess this one is really outdated already. If not, please, v rebase and submit a new pr.

**Pull request closed without merging**